es-split/main.go

```diff
@@ -0,0 +1,201 @@
+package main
```

Comment:
I see 2 better options here :

- merge the following logic with `astits/main.go`
- create a `cmd` folder --> move `es-split` in it and rename it `astits-es-split` --> create a `astits-probe` folder in the `cmd folder` and move `astits/main.go` inside it

Reply:
Will do the second option

Reply:
Done